### ISSUE TITLE
Accessibility: Unable to use  VoiceOver navigation to re-enter application content

### ISSUE THREAD

**Issue by jekis913** [desktop/desktop#10303]
### Describe the bug

Using VoiceOver on macOS, If I stop interacting with the content area of Github Desktop, I am not able to re-enter the application to continue working. The only solution is closing and reopening the app.

### Version & OS

- Desktop version: 2.5.4-beta1
- macOS version: 10.15.6

### Steps to reproduce the behavior

1. Launch VoiceOver (press command + f5)
2. Open Desktop
3. Start interacting with Desktop using VoiceOver navigation by pressing control + option + shift + 🔽 
4. Stop interacting with Desktop by pressing control + option + shift + 🔼 until you reach the top of the Window
5. Try to resume interacting with Desktop by pressing control + option + shift + 🔽
6. Note how you cannot navigate back into Desktop

### Expected behavior

Voiceover should be able to see and re-enter the content area.

### Actual behavior

Voiceover can only access the close, minimize and full screen buttons. The content of the app is unreachable.

### Additional context

This This issue was tracked as electron/electron#17207. The discord developers back ported the fix to electron 7.1 in discord/electron#6. There is also what seems to be a better solution in electron/electron#24699, which was shipped in version 9.1.2.

**Comment by rafeca** [comment]
Thanks for filing this issue! ✨

We're in the process of upgrading to electron v8, which as you mention should fix this issue (see: https://github.com/desktop/desktop/pull/9766).  Our estimation is that the electron upgrade will get included in the next Desktop release.

We're going to close this issue once the electron PR gets merged.
